### ISSUE TITLE
check new logging options in OpenLDAP 2.6

### ISSUE THREAD

**Issue by davidcoutadeur** [ltb-project/openldap-rpm#84]
new 2.6 version relies on logfile directive
See ITS#6949 https://bugs.openldap.org/show_bug.cgi?id=6949

**Comment by davidcoutadeur** [comment]
For information, see this config example: (taken from https://bugs.openldap.org/show_bug.cgi?id=6949)

```
ldapmodify -x -H ldapi:/// -D cn=config -w secret
dn: cn=config
changetype: modify
add: olcLogFile
olcLogFile: /var/symas/slapd.log
-
add: olcLogFileOnly
olcLogFileOnly: TRUE
-
add: olcLogFileRotate
olcLogFileRotate: 12 10 1
```

**Comment by davidcoutadeur** [comment]
Integrated in slapd-cli 2.9, included in openldap LTB 2.6.2 release
